lib/WorldState.js

```diff
@@ -199,9 +199,19 @@ module.exports = class WorldState {
     /**
      * The state of the Void Trader
      * @type {VoidTrader}
+     * @deprecated
      */
     [this.voidTrader] = parseArray(deps.VoidTrader, data.VoidTraders, deps);
```

Comment:
Delete this so we don't pause the array twice, and move the docs down to the later destructure